marsha/core/admin.py

```diff
@@ -151,3 +152,12 @@ class PlaylistAdmin(BaseModelAdmin):
     list_display = ("title", "organization", "created_by", "is_public")
     exclude = ("duplicated_from",)
     inlines = [VideosInline, PlaylistAccessesInline]
+
+
+@admin.register(LTIPassport, site=admin_site)
+class LTIPassportAdmin(BaseModelAdmin):
+    """Admin class for the LTIPassport model."""
+
+    list_display = ("created_on", "consumer_site", "playlist", "is_enabled")
+    readonly_fields = ["oauth_consumer_key", "shared_secret"]
```

Comment:
"created_on" should also be readonly? Anyway, I think these 3 fields would still be readonly even if you don't list them here?

Reply:
created_on is not displayed in detail view, should it ?
I used it in this list because it can be usefull information, but mostly because we need something to click  on if playlist and consumer_site are empty

Reply:
Oh ok. I think it's good to have it in the detail view.